WooCommerce/Classes/System/DefaultFeatureFlagService.swift

```diff
@@ -1,18 +1,22 @@
 struct DefaultFeatureFlagService: FeatureFlagService {
     func isFeatureFlagEnabled(_ featureFlag: FeatureFlag) -> Bool {
+        let buildConfig = BuildConfiguration.current
+
         switch featureFlag {
         case .barcodeScanner:
-            return BuildConfiguration.current == .localDeveloper || BuildConfiguration.current == .alpha
+            return buildConfig == .localDeveloper || buildConfig == .alpha
         case .editProducts:
             return true
         case .editProductsRelease2:
             return true
         case .editProductsRelease3:
-            return BuildConfiguration.current == .localDeveloper || BuildConfiguration.current == .alpha
+            return buildConfig == .localDeveloper || buildConfig == .alpha
         case .readonlyProductVariants:
             return true
         case .refunds:
             return true
+        case .inAppFeedback:
```

Comment:
nit: it looks like the cases are currently sorted by name, might be nice to keep the order? 😄

Reply:
OMG. You don't miss anything. 😄 Fixed in 041c954e670ecebb88025e7e140bef8d68bca411.